Bottom: 7cb237f86f79fb7f101ef7e333425586fa4a14ea
Top:    b88d986fce8f7d2a3fec89f874e2ed800125b46d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-24 13:39:27 +0000

Win32 FIX


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index f736c54..f1018b0 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -28,6 +28,10 @@
 #   include <GL/gl.h>
 #endif
 
+#ifdef WIN32
+#   include "hid/common/glext.h"
+#endif
+
 #include <gtk/gtkgl.h>
 #include "hid/common/hidgl.h"
